internal/nodes/avi_model_nodes.go

```diff
@@ -253,12 +259,16 @@ func (o *AviObjectGraph) RemovePGNodeRefs(pgName string, vsNode *AviVsNode) {
 
 }
 
-func (o *AviObjectGraph) RemoveHTTPRefsFromSni(httpPol, hppMap string, sniNode *AviVsNode) {
-
+func (o *AviObjectGraph) RemoveHTTPRefsStringGroupsFromSni(httpPol, hppMap string, sniNode *AviVsNode) {
+	var stringGroupToRemove []string
 	for i, pol := range sniNode.HttpPolicyRefs {
 		if pol.Name == httpPol {
 			for j, hppmap := range sniNode.HttpPolicyRefs[i].HppMap {
 				if hppmap.Name == hppMap {
+					if len(sniNode.HttpPolicyRefs[i].HppMap[j].StringGroupRefs) > 0 {
+						sgref := strings.Split(sniNode.HttpPolicyRefs[i].HppMap[j].StringGroupRefs[0], "=")[1]
```

Comment:
Same as above

Reply:
done

Reply:
I cannot see anychange

Reply:
The change is done and also visible in the Files changed.